examples/word_finder.md

```diff
@@ -0,0 +1,185 @@
+# Building a Word Finder App with Deno
+
+## Getting Started
```

Comment:
I'm still not sure about this example.  It doesn't get some of the fundamental position of the core technology support of Deno right.  I personally would never suggest to anyone to write an app like this.  @lucacasonato thoughts?

Reply:
if there are other examples that you think are more relevant I'm happy to write them.